scripts/fetch_earthquakes.sh

```diff
@@ -9,14 +9,14 @@ ENDTIME=$(date +'%FT%H:%M:%S')
 FILENAME="earthquakes_last_90d.txt"
 STARTTIME=$(date -d '-90 days' +'%FT%H:%M:%S')
 
-curl --fail "http://arclink.ethz.ch/fdsnws/event/1/query?starttime=$STARTTIME&endtime=$ENDTIME&minmagnitude=1&format=text&nodata=404" > $FILENAME
+curl --fail "http://arclink.ethz.ch/fdsnws/event/1/query?starttime=$STARTTIME&endtime=$ENDTIME&minmagnitude=1&format=text&nodata=404&&EventType=earthquake" > $FILENAME
```

Comment:
you have double `&&`, same for the one bellow

Reply:
oups thanks, fixed